configure.ac

```diff
@@ -1721,7 +1721,7 @@ if test -z "$ssl_backends" -o "x$OPT_SSL" != xno &&
       AC_MSG_NOTICE([pkg-config: SSL_LDFLAGS: "$SSL_LDFLAGS"])
       AC_MSG_NOTICE([pkg-config: SSL_CPPFLAGS: "$SSL_CPPFLAGS"])
 
-      LIB_OPENSSL=`echo $SSL_LDFLAGS | sed -e 's/-L//g'`
+      LIB_OPENSSL=`echo $SSL_LDFLAGS | sed -e 's/^-L//'`
```

Comment:
Maybe `$SED` for consistency, but honestly it probably does not matter too much given the simple pattern and the widespread use of `sed` elsewhere (e.g. acinclude.m4).

Reply:
Right, and since this check has worked like this for so long for so many users that I rather change as little as possible...